### PR TITLE
[14.0][MIG] utm

### PR DESCRIPTION
**Pull request by RaulOForgeFlow**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: openupgrade_scripts/scripts/utm/14.0.1.0/upgrade_analysis.txt [hunk 1]
```diff
@@ -12,4 +12,4 @@ NEW utm.source: utm.utm_source_linkedin
 NEW utm.source: utm.utm_source_monster
 DEL utm.stage: utm.campaign_stage_1
 DEL utm.stage: utm.campaign_stage_2
-DEL utm.stage: utm.campaign_stage_3
+DEL utm.stage: utm.campaign_stage_3
```
Comment (MiquelRForgeFlow) [inline openupgrade_scripts/scripts/utm/14.0.1.0/upgrade_analysis.txt]: I don't see what you changed here. Can you try to revert it?

File: openupgrade_scripts/scripts/utm/14.0.1.0/upgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+DEL utm.campaign: utm.utm_campaign_email_campaign_products
+DEL utm.campaign: utm.utm_campaign_email_campaign_services
+DEL utm.campaign: utm.utm_campaign_fall_drive
+DEL utm.stage: utm.campaign_stage_1
+DEL utm.stage: utm.campaign_stage_2
+DEL utm.stage: utm.campaign_stage_3
+# NOTHING TO DO: noupdate = "0" records. Sample campaigns are now in utm_demo.xml
+
+
+
```
Comment (MiquelRForgeFlow) [inline openupgrade_scripts/scripts/utm/14.0.1.0/upgrade_analysis_work.txt]: too much blank lines

**Review by flotho — APPROVED** [review]
AFAIK, looks good to me as there's mostly nothing to DO ;-)

**Review by jcdrubay — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I have added a script for removing (if possible) that demo data not put as demo data in the previous version.

**Pull request merged**